src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/DataprocDAO.scala

```diff
@@ -27,4 +28,8 @@ trait DataprocDAO {
   def getClusterMasterInstanceIp(googleProject: GoogleProject, clusterName: String)(implicit executionContext: ExecutionContext): Future[Option[String]]
 
   def getClusterErrorDetails(operationName: String)(implicit executionContext: ExecutionContext): Future[Option[ClusterErrorDetails]]
+
+  def deleteClusterInitBucket(googleProject: GoogleProject, clusterName: String)(implicit executionContext: ExecutionContext): Future[Option[String]]
+
+  def deleteBucket(googleProject: GoogleProject, bucketName: String)(implicit executionContext: ExecutionContext): Future[Unit]
```

Comment:
I was thinking that it would be nice to separate the Storage and Dataproc into separate DAO's, and also move those DAO's to `workbench-libs/google`. That work should be its own ticket though.